src/nodes/contributor/contributor_view.js

```diff
@@ -0,0 +1,175 @@
+var $$ = require("lens/substance/application").$$;
+var _ = require("underscore");
+
+var AMSContributorView = function(node, viewFactory) {
+  ContributorView.call(this, node, viewFactory);
+};
+
+var LABELS = {
+  "amsrefs": "AMSRefs",
+  "bibtex": "BibTeX"
+};
+
+AMSContributorView.Prototype = function() {
+  this.renderBody = function() {
+    // Contributor Name
+    // -------
+
+    this.content.appendChild($$('.contributor-name', {text: this.node.name}));
+
+    // Contributor Role
+    // -------
+
+    if (this.node.role) {
+      this.content.appendChild($$('.role', {text: this.node.role}));
+    }
+
+
+    // Add Affiliations
+    // -------
+
+    this.content.appendChild($$('.affiliations', {
+      children: _.map(this.node.getAffiliations(), function(aff) {
+
+        var affText = _.compact([
+          aff.department,
+          aff.institution,
+          aff.city,
+          aff.country
+        ]).join(', ');
+
+        return $$('.affiliation', {text: affText});
+      })
+    }));
+
+
+
+    // Present Address
+    // -------
+
+    if (this.node.present_address) {
+      this.content.appendChild($$('.label', {text: 'Present address'}));
+      this.content.appendChild($$('.contribution', {text: this.node.present_address}));
+    }
+
+    // Contribution
+    // -------
+
+    if (this.node.contribution) {
+      this.content.appendChild($$('.label', {text: 'Contribution'}));
+      this.content.appendChild($$('.contribution', {text: this.node.contribution}));
+    }
+
+    // Equal contribution
+    // -------
+
+    if (this.node.equal_contrib && this.node.equal_contrib.length > 0) {
+      this.content.appendChild($$('.label', {text: 'Contributed equally with'}));
+      this.content.appendChild($$('.equal-contribution', {text: this.node.equal_contrib}));
+    }
+
+
+    // Emails
+    // -------
+
+    if (this.node.emails.length > 0) {
+      this.content.appendChild($$('.label', {text: 'For correspondence'}));
+      this.content.appendChild($$('.emails', {
+        children: _.map(this.node.emails, function(email) {
+          return $$('a', {href: "mailto:"+email, text: email});
+        })
+      }));
+    }
+
+
+    // Funding
+    // -------
+
+    if (this.node.fundings.length > 0) {
+      this.content.appendChild($$('.label', {text: 'Funding'}));
+      this.content.appendChild($$('.fundings', {
+        children: _.map(this.node.fundings, function(funding) {
+          return $$('.funding', {text: funding});
+        })
+      }));
+    }
+
+
+    // Competing interests
+    // -------
+
+    if (this.node.competing_interests.length > 0) {
+      this.content.appendChild($$('.label', {text: 'Competing Interests'}));
+      this.content.appendChild($$('.competing-interests', {
+        children: _.map(this.node.competing_interests, function(ci) {
+          return $$('.conflict', {text: ci});
+        })
+      }));
+    }
+
+    // MR if available
+    // -------
+
+    if (this.node.properties.mrauth) {
```

Comment:
Let me try that again, @michael. I meant this part. It seems I should be using `this.node.mrauth` in this blog but that doesn't exist when I try. I'm guessing I have to declare the label somewhere so that it's pulled out of the properties but I couldn't figure out how yet.